sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### 4.5.2b4 (Unreleased)
 
 #### Features Added
+* Added conditional support for preview feature of Computed Properties on Python SDK. See [PR 33626](https://github.com/Azure/azure-sdk-for-python/pull/33626)
```

Comment:
What's it conditional on? Or does this just mean it's still a service preview feature?
Does the service plan to GA this feature before the next SDK GA?

Reply:
This is supposed to mean that it is still a preview feature that needs to be enabled on the account. As of now, I do not know if they plan to GA this feature before the next SDK GA.

Reply:
we can just rename this to match other feature entries that are in preview like priority based throttling:
<img width="996" alt="image" src="https://github.com/Azure/azure-sdk-for-python/assets/30335873/5b781ee1-4f17-4a59-aff0-382ab6d54f59">

Reply:
I went ahead and changed the wording of the changelog, but I will omit the feature doc link in the changelog, it seems the pipeline doesn't like that link and labels it as broken. I'll include it instead in the PR description.